packages/primitives-core/src/css.js

```diff
@@ -44,9 +44,11 @@ function handleInterpolation(interpolation: *, i: number, arr: Array<*>) {
     return
   }
   let isIrrelevant = interpolation == null || type === 'boolean'
-  let isRnStyle =
-    (type === 'object' && !Array.isArray(interpolation)) || type === 'number'
-  if (lastType === 'string' && (isRnStyle || isIrrelevant)) {
+  let isRnStyle = type === 'object' && !Array.isArray(interpolation)
+  if (
+    (lastType === 'string' || lastType === 'number') &&
+    (isRnStyle || isIrrelevant)
+  ) {
```

Comment:
Hi. Sorry for the very late response. I am now picking up this PR again. 

Before I do so, can I ask, do we want to support older RN versions? RNW has [released 0.18](https://github.com/necolas/react-native-web/releases/tag/0.18.0) which allows us to fix the number interpolation issue. 

1. Can we require RNW >= 0.18, or should we also be supporting older versions? 
2. If we require RNW >= 0.18, will that require an emotion major version bump?

(I will add some additional changes before addressing your other comments.)

Reply:
That's a good question... for which I don't have a good answer. All of the RN-related deps are problematic when it comes to versioning because they move at a vastly different pace than, let's say, React itself.

Ideally, we wouldn't break older versions supported here as that would be a breaking change. Is this possible though? Can we feature-detect a version?